tests/test_operation.py

```diff
@@ -1234,10 +1234,23 @@ def test_arithmetic_errors(self):
 class TestDecomposition:
     """Test for operation decomposition"""
 
+    def test_decomposition_with_int_wire(self):
+        """Test that the decomposition of a non-parametrized Operation works
+        when we simply pass an integer as a wire without further specification."""
+        decomp_1 = qml.Hadamard.decomposition(1)
+        decomp_2 = qml.Hadamard.decomposition([1])
+        expected_decomp = qml.Hadamard.decomposition(wires=1)
```

Comment:
Sure I can try that - very odd, though, since there is a test in line 1004 of `test_parametric_ops` that checks this, but I suppose it calls `.decompose` instead of `.decomposition` explicitly on the class.